src/components/KTable/KTable.vue

```diff
@@ -139,7 +139,7 @@
             :key="`k-table-${tableId}-row-${rowIndex}`"
             :tabindex="isClickable ? 0 : null"
             :role="isClickable ? 'link' : null"
-            v-on="hasSideBorder ? tdlisteners(row, row) : null"
+            v-on="hasSideBorder ? tdlisteners(row, row) : {}"
```

Comment:
question: could you do `undefined` here instead? I know `null` probably wasn't valid for Vue 3

Reply:
I tested with `undefined` but unfortunately the warning persists. Perhaps Vue only expects object values: [source](https://github.com/vuejs/core/blob/9c304bfe7942a20264235865b4bb5f6e53fdee0d/packages/runtime-core/src/helpers/toHandlers.ts#L10)